content/browser/gpu/gpu_data_manager_impl_private.cc

```diff
@@ -555,6 +555,10 @@ void GpuDataManagerImplPrivate::InitializeGpuModes() {
 #endif  // BUILDFLAG(IS_FUCHSIA)
   }
 
+// hack
+  fallback_modes_.clear();
+  fallback_modes_.push_back(gpu::GpuMode::HARDWARE_GL);
+
```

Comment:
This might be needed because of l.553 -- we should be taking  
the #else in l.551 (to be confirmed), and we end up using  
vulkan for some reason. There is a build flag to enable/disable  
vulkan [1] and this is probably enabled because it thinks we  
are on Linux [2] (and we should force disable it for is_starboard).


[1] https://source.corp.google.com/h/lbshell-internal/cobalt_src/+/main:content/browser/gpu/gpu_data_manager_impl_private.cc;l=402-416?q=vulkanallowed&ss=h%2Flbshell-internal%2Fcobalt_src%2F%2B%2Frefs%2Fheads%2Fmain&start=1
[2] https://source.corp.google.com/h/lbshell-internal/cobalt_src/+/main:gpu/vulkan/features.gni;l=11?q=vulkan%20file:%5C.gni&ss=h%2Flbshell-internal%2Fcobalt_src%2F%2B%2Frefs%2Fheads%2Fmain

Reply:
Maybe https://github.com/youtube/cobalt/pull/4361 will help removing this delta.